aspnetcore/fundamentals/minimal-apis/includes/route-handlers.md

```diff
@@ -1,4 +1,4 @@
-Route handlers are methods that execute when the route matches. Route handlers can be a function of any shape, including synchronous or asynchronous. Route handlers can be a lambda expression, a local function, an instance method or a static method.
+Route handlers are methods that execute when the route matches. Route handlers can be a function of any type, including synchronous or asynchronous. Route handlers can be a lambda expression, a local function, an instance method or a static method.
```

Comment:
Shape in jargon for signature. I wonder if one of the following would be better

1. Route handlers can be a function of any signature,
2. Route handlers can be a function of any signature or structure,

@david-acker @tdykstra @brunolins16 anyone have a better suggestion?

Reply:
With all thte examples I don't think "shape" is necessary:

```suggestion
Route handlers are methods that execute when the route matches. Route handlers can be a lambda expression, a local function, an instance method or a static method. Route handlers can be synchronous or asynchronous. 
```